auctioneer/program/src/sell/config.rs

```diff
@@ -2,7 +2,7 @@ use anchor_lang::prelude::*;
 use solana_program::clock::UnixTimestamp;
 
 pub const BID_SIZE: usize = 8 + 1 + 32;
-pub const LISTING_CONFIG_SIZE: usize = 8 + 1 + 8 + 8 + BID_SIZE + 1;
+pub const LISTING_CONFIG_SIZE: usize = 8 + 1 + 8 + 8 + BID_SIZE + 1 + 8;
```

Comment:
It would, but my intention is to knock out all of these tickets this week before anyone really starts using Auctioneer. I guess I could add some padding as well so this week's changes don't break anything.